log4stash.nuspec

```diff
@@ -36,17 +36,19 @@
     <copyright>Copyright 2017</copyright>
     <tags>log logging exception elasticsearch log4net appender logstash filters</tags>
     <dependencies>
-		<group targetFramework=".NETFramework3.5">
-			<dependency id="log4net" version="2.0.8.0" />
-		</group>
+		
 		<group targetFramework=".NETFramework4.0">
 			<dependency id="log4net" version="2.0.8.0" />
+			<dependency id="RestSharp" version="105.2.3" />
+			<dependency id="Microsoft.Bcl.Async" version="1.0.168" />
 		</group>
 		<group targetFramework=".NETFramework4.5">
 			<dependency id="log4net" version="2.0.8.0" />
+			<dependency id="RestSharp" version="105.2.3" />
```

Comment:
Microsoft.Bcl.Async ?

Reply:
It's required to support proper async in .net 4.